vllm/model_executor/parameter.py

```diff
@@ -0,0 +1,160 @@
+from typing import Callable, List, Optional, Tuple, Union
+
+import torch
+from torch.nn import Parameter
+
+from vllm.logger import init_logger
+
+__all__ = [
+    "vLLMParameter", "PackedvLLMParameter", "ScalerToArrayvLLMParameter"
+]
+
+logger = init_logger(__name__)
+
+
+class vLLMParameter(Parameter):
+
+    def __new__(cls,
+                data: torch.Tensor,
+                requires_grad: Optional[bool] = False,
+                **kwargs):
+
+        return super().__new__(cls, data=data, requires_grad=requires_grad)
+
+    def __init__(self,
+                 data: torch.Tensor,
+                 requires_grad: Optional[bool] = False,
```

Comment:
`requires_grad` will always be false, so let's just set it to False everywhere + remove the argument